src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/models/PartyDetailsContent.java

```diff
@@ -0,0 +1,82 @@
+package uk.gov.hmcts.cmc.claimstore.documents.content.models;
+
+import uk.gov.hmcts.cmc.domain.models.Address;
+
+public class PartyDetailsContent {
+
+    private final String type;
+    private final String fullName;
+    private final boolean nameAmended;
+    private final String businessName;
+    private final String contactPerson;
+    private final Address address;
+    private final boolean addressAmended;
+    private final Address correspondenceAddress;
+    private final String dateOfBirth;
+    private final String email;
+
+    @SuppressWarnings("squid:S00107") // Number of method parameters
```

Comment:
comments should be why not what.
REC
Something like:
> Allow more method parameters - Content providers are formatted values and aren't worth splitting into multiple models.

if you wanted to fix the warning you could create something like
`ContactDetailsContent`
with the address details and email on it